src/message.ts

```diff
@@ -810,7 +965,19 @@ export class MediaMessage extends Message {
   }
 
   /**
-   * @todo document me
+   * Get the MediaMessage filename, etc: `how to build a chatbot.pdf`..
+   *
+   * @returns {string}
+   * @example
+   * ```ts
```

Comment:
Could you find a button with `resolve` text on it, near this review message?

Please help me to find it, because @binsee also need that button... thanks.